.github/ISSUE_TEMPLATE/2-PROXY.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Problem with the client
+about: The client is printing a stack trace or failing unexpectedly.
+
+---
+
+# Environment
+
+- `pub version` or `flutter packages pub version`:
+- OS version:
+- Are there any firewalls or proxies that could be impacting the problem?
+
+# Problem
+
+<!--
+  Please describe the problem in detail, including the specific command that is
+  failing.
+-->
+
+## Expected behavior
+
+<!--
+  What did you expect the result of the command to be?
+-->
+
+## Actual behavior
+
+<!--
+  What actually happened?
+-->
+
+## --trace output
```

Comment:
Add more details here? The full command to run...

Reply:
I mention in the "Problem" section to put the command...